common/ayon_common/distribution/control.py

```diff
@@ -237,6 +237,30 @@ def _wait_for_other_process(
     yield output
 
 
+def change_permissions_recursive(path: str, mode: Optional[int] = None):
+    if IS_WINDOWS or not os.path.exists(path):
+        return
+
+    if mode is None:
+        mode = 0o777
```

Comment:
Couldn't `0o777` just be the default value in function signature?

Reply:
It is actually default value if you look into the Python implementation, I don't know why it does not apply?